menuconfig.py

```diff
@@ -165,100 +165,276 @@
 view the help of the selected item without leaving the dialog.
 """[1:-1].split("\n")
 
-def _init_styles():
-    global _PATH_STYLE
-    global _SEPARATOR_STYLE
-    global _LIST_STYLE
-    global _LIST_SEL_STYLE
-    global _LIST_INVISIBLE_STYLE
-    global _LIST_INVISIBLE_SEL_STYLE
-    global _HELP_STYLE
-    global _DIALOG_FRAME_STYLE
-    global _DIALOG_BODY_STYLE
-    global _DIALOG_EDIT_STYLE
-    global _JUMP_TO_EDIT_STYLE
-    global _INFO_TEXT_STYLE
+# Default menuconfig theme
+DEFAULT_STYLE = \
+{
+        # Top row in the main display, with the menu path
+        "path":             "bold black on white",
+        # Separator lines between windows. Also used for the top line in the symbol
+        "separator":        "bold black on yellow",
+        # List of items, e.g. the main display
+        "list":             "black on white",
+        # Style for the selected item
+        "selection":        "bold white on blue",
+        # Like _LIST_(SEL_)STYLE, for invisible items. Used in show-all mode.
+        "inv_list":         "red on white",
+        "inv_selection":    "red on white",
+        # Help text windows at the bottom of various fullscreen dialogs
+        "help":             "bold black on white",
+        # Frame around dialog boxes
+        "frame":            "bold black on yellow",
+        # Body of dialog boxes
+        "body":             "white on black",
+        # Edit box in pop-up dialogs
+        "edit":             "white on blue",
+        # Edit box in jump-to dialog
+        "jump_to_edit":     "white on blue",
+        # Symbol information text
+        "text":             "black on white"
```

Comment:
Instead of having `_DEFAULT_STYLE` be a dictionary, I wonder if it could just be a string, in the normal `MENUCONFIG_STYLE` format. Everything would then go through a single style parsing code path.

The only difference for the default style would be that it's hardcoded to be parsed before `MENUCONFIG_STYLE`. This could be done by just adding `default` or the like to the beginning of the `MENUCONFIG_STYLE` string in the code, before parsing it.